src/components/ContentCard.tsx

```diff
@@ -40,9 +41,16 @@ const ContentCard: FC<ContentCardProps> = ({
   upkeepLevel,
   links,
   details,
+  hasArrow,
 }) => {
   return (
-    <div className="bg-green-100 rounded-md">
+    <div
+      className={
+        hasArrow
+          ? "bg-green-100 rounded-md hover:bg-green-200"
```

Comment:
would love @thansidwell's input too, but would we be able to add a `transition-colors duration-[250ms]` here? this would make the hover fade animate in and out with the same duration as the next-ui buttons

the hover i'm talking about:

https://github.com/CodeForPhilly/vacant-lots-proj/assets/8061917/bde45f65-3201-4121-a9ff-49b75664b234